go/runtime/host/sandbox/sandbox.go

```diff
@@ -126,6 +128,10 @@ type sandboxedRuntime struct {
 	stopCh    chan struct{}
 	ctrlCh    chan interface{}
 
+	// generation is the generation number of the underlying process. It gets incremented by one on
```

Comment:
Oh, good point, that seems cleaner! Will change it.

Reply:
In theory, both approaches can still queue requests for old generations, if you put huge timeouts in `Abort` (e.g. before `r.generation.Load()` and `case r.ctrlCh <- &abortRequest`). But not sure if in practice this can be a problem. Ideally, abort requests would contain the ID of the thing they are trying to abort, I guess :thinking:

Reply:
Yes, I was thinking about that. The ID (or generation) would ideally be something returned from a previous `Call`. The problem is also that currently a bunch of requests can be implicitly queued in the stack in case the channel is full.

Reply:
> The ID (or generation) would ideally be something returned from a previous Call.

Yes, that would probably be the right way to go. 

> The problem is also that currently a bunch of requests can be implicitly queued in the stack in case the channel is full.

But we don't need to go that way, as both solutions (generations and clearing) should solve the problem.

Reply:
Pushed a version that drains the channel.